shapely/tests/test_io.py

```diff
@@ -88,51 +88,6 @@
 ]
 
 
-class ShapelyGeometryMock:
-    def __init__(self, g):
-        self.g = g
-        self.__geom__ = g._ptr if hasattr(g, "_ptr") else g
```

Comment:
This was some leftover code from pygeos (testing it could convert shapely geometries), that could be cleaned-up (while removing all references of `__geom__`)